great-idea/css/index.css

```diff
@@ -63,4 +63,234 @@ h1, h2, h3, h4, h5 {
+	position: fixed;
+	background-color: white;
+	width: 900px;
+	z-index: 5;*/
+}
+
+
+body {
+	margin: 0 auto;
+	width: 880px;
+	font-size: 16px;
+}
+
+header {
+	background-color: white;
+	width: 100%;
+	align-items: center;
+	height: 36px;
+}
+
+header nav a {
+	vertical-align: middle;
+	margin: 10px 28px;
+	font-size: 15px;
+	text-decoration: none;
+	background-color: white;
+	color: lightgrey;
+}
+
+header nav a:first-of-type {
+	margin-left: 0;
+}
+
+header nav a:hover {
+	color: black;
+	font-size: 18px;
+	border-bottom: 2px dashed black;
+}
+
+
+.logo {
+	padding-left: 25px;
+	background-color: white;
+	/*justify-content: flex-end;*/
+	margin: 0 0 0 0;
+	vertical-align: middle;
+}
+
+h1 {
+	font-size: 70px;
+	text-align: center;
+}
+
+h3 {
+	font-size: 26px;
+	margin-bottom: 14px;
+}
+
+
+
+.first-section {
+	margin-top: 45px;
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+}
+
+.first-section button {
+	border: 1px solid black;
+	background-color: white;
+	padding: 5px 40px;
+	font-family: 'Titillium Web', sans-serif;
+	font-weight: 600;
+	font-size: 16px;
+	display: flex;
+	text-align: center;
+	align-items: center;
+	margin-left: 70px;
+}
+
+.left-side {
+	padding-top: 50px;
+}
+
+.second-section {
+	padding: 30px 0 50px 0;
+	margin-top: 50px;
+	border-top: 1px solid black;
+	display: flex;
+	justify-content: space-between;
+}
+
+
+.middle-img {
+	width: 880px;
+}
+
+.third-group {
+	padding-top: 45px;
+	padding-bottom: 50px;
+	border-bottom: 1px solid black;
+	display: flex;
+	justify-content: space-between;
+	padding-right: 15px;
+}
+
+.third-section:last-of-type {
+	margin-right: 0;
+}
+/*.third-section {
+	width: 275px;
+	margin-right: 15px;
+	display: inline-block;
+}*/
+
+footer .footer {
+	margin-top: 30px;
+	margin-bottom: 40px;
+}
+
+footer .copyright {
+	text-align: center;
+	margin-top: 40px 0;
+}
+
+.services-header-img {
+	padding-bottom: 30px;
+	padding-top: 5px;
+}
+
+.services-heading {
+	border-bottom: 1px solid black;
+	padding-bottom: 35px;
+}
+
+.services-boxes {
+	border-bottom: 2px solid black;
+	margin-bottom: 40px;
+}
+
+.services-boxes .container1 {
+	display: flex;
+	justify-content: space-between;
+	margin-top: 40px;
+}
+
+.services-boxes .container1 .boxes {
+	width: 420px;
+	border: 1px solid black;
+	display: flex;
+	flex-direction: column;
+	background: #F4F4F4;
+	padding: 25px;
+	}
+
+.services-boxes .container1 button {
+	border-radius: 5px;
+	border: 1px solid black;
+	font-family: inherit;
+	background: white;
+	padding: 5px 25px 5px;
+	width: 120px;
+	margin-top: 20px;
+}
+.services-boxes .container1 .last-boxes {
+	margin-bottom: 40px;
+}
+
+section .container2 {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 80px;
+}
+
+section .container2 .list-side {
+	display: flex;
+	flex-direction: column;
+	margin-right: 50px;
+}
+section .container2 .list-side ul {
+	display: flex;
+	flex-direction: column;
+}
+
+section .container2 .list-side ul li {
```

Comment:
really good use grabbing this nested li, it makes it a lot easier on yourself and css when you grab them like this.. would like to see a lot of your styles written like this.